### PR TITLE
Changelog for v2.155.0

### PR DESCRIPTION
**Pull request by vEpiphyte**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -6,6 +6,116 @@
...
+- Allow taxonomy prefix lift and filter operations to work with taxon parts.
+  (`#3329 <https://github.com/vertexproject/synapse/pull/3329>`_)
+- Update the allowed versions of the ``cbor2``, ``pycryptodome``,
+  ``pygments``, ``vcrpy``, and ``xxhash`` libraries. Update the pinned version
+  of the ``lark`` library.
+  (`#3418 <https://github.com/vertexproject/synapse/pull/3418>`_)
+
+Bugfixes
+--------
+- Fix a regression in graph projection for computing large graphs in Storm.
```
Comment (invisig0th) [inline CHANGELOG.rst:106]: performance regression

File: CHANGELOG.rst [hunk 2]
```diff
@@ -6,6 +6,116 @@
...
+
+  ``it:av:filehit``
+    Please use ``it:av:scan:result``.
+
+  ``it:av:prochit``
+    Please use ``it:av:scan:result``.
+
+Features and Enhancements
+-------------------------
+
```
Comment (Cisphyx) [inline CHANGELOG.rst:80]: extra blank line

File: CHANGELOG.rst [hunk 3]
```diff
@@ -6,6 +6,116 @@
...
+- Add a ``detach()`` method to the Storm ``view`` object. This will detach a
+  forked View from its parent.
+  (`#3323 <https://github.com/vertexproject/synapse/pull/3323>`_)
+- Change the method used to generate the ``took`` value in the Storm ``fini``
+  message to use a monotonic clock.
+  (`#3325 <https://github.com/vertexproject/synapse/pull/3325>`_)
+- Performing a invalid pivot in operation with a form target
```
Comment (Cisphyx) [inline CHANGELOG.rst:86]:
```suggestion
- Performing an invalid "pivot in" operation with a form target
```

File: CHANGELOG.rst [hunk 4]
```diff
@@ -6,6 +6,116 @@
...
+- Update the allowed versions of the ``cbor2``, ``pycryptodome``,
+  ``pygments``, ``vcrpy``, and ``xxhash`` libraries. Update the pinned version
+  of the ``lark`` library.
+  (`#3418 <https://github.com/vertexproject/synapse/pull/3418>`_)
+
+Bugfixes
+--------
+- Fix a regression in graph projection for computing large graphs in Storm.
+  (`#3375 <https://github.com/vertexproject/synapse/pull/3375>`_)
+- Fix a conflict between Storm ``$lib.inet.http`` fucnctions and ``vcrpy``
```
Comment (Cisphyx) [inline CHANGELOG.rst:108]:
```suggestion
- Fix a conflict between Storm ``$lib.inet.http`` functions and ``vcrpy``
```

**Review by Cisphyx — APPROVED** [review]

**Pull request merged**